src/views/33Together/33together.scss

```diff
@@ -47,6 +47,10 @@
   }
 }
 
+.light .vegas {
+  filter: invert(0.99);
+}
+
```

Comment:
is this how you made that vegas-container show better on light mode?

Reply:
Yeee, just threw the theme name into app container so now we can have custom light/dark styles as needed